Assets/Scripts/Models/Character/Character.cs

```diff
@@ -577,4 +582,18 @@ private void LoadStats()
 
         Debug.ULogChannel("Character", "Initialized " + stats.Count + " Stats.");
     }
+
+    /// <summary>
+    /// Use the stats of the character to determine various traits.
+    /// </summary>
+    private void UseStats()
+    {
+        // Base character speed on their dexterity.
+        speed = Convert.ToSingle(stats["Dexterity"].Value) / 2;
+
```

Comment:
That's part of the reason i wanted feedback. Do we want characters to be able to level up or other such things. CAN their dexterity change? and this part of the code was just initializing on character creation. will add more notes.